test/benchmarks/passes.py

```diff
@@ -188,3 +194,61 @@ def setup(self, n_qubits, depth, _):
 
     def time_collect_multiq_block(self, _, __, max_block_size):
         CollectMultiQBlocks(max_block_size).run(self.dag)
+
+
+class LitinskiTransformationPassQFTBenchmarks:
+    params = [5, 10, 100, 1000]
+
+    param_names = ["n_qubits"]
+    timeout = 300
+
+    def setup(self, n_qubits):
+        target = Target.from_configuration(["rz"] + get_clifford_gate_names(), n_qubits)
+        qft = synth_qft_full(n_qubits, do_swaps=False)
+        tqft = transpile(qft, target=target, seed_transpiler=0)
+        self.dag = circuit_to_dag(tqft)
+
+    def time_litinski_transformation_fix_clifford(self, _):
+        _pass = LitinskiTransformation(fix_clifford=True)
+        _pass.run(self.dag)
+
+    def time_litinski_transformation_no_fix_clifford(self, _):
+        _pass = LitinskiTransformation(fix_clifford=False)
+        _pass.run(self.dag)
+
+
+class LitinskiTransformationPassQasmBenchmarks:
+
+    def build_from_qasm(self, benchmark_name):
+        full_name = os.path.join(self.qasm_dir, benchmark_name)
+        qc = QuantumCircuit.from_qasm_file(full_name)
+        transpiled_qc = transpile(qc, basis_gates=self.basis_gates, seed_transpiler=0)
+        transpiled_dag = circuit_to_dag(transpiled_qc)
+        return transpiled_dag
+
+    def setup(self):
+        self.qasm_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), "qasm")
+        self.basis_gates = ["rz"] + get_clifford_gate_names()
+
+        self.qft = self.build_from_qasm("qft_N100.qasm")
+        self.square_heisenberg = self.build_from_qasm("square_heisenberg_N100.qasm")
+        self.qaoa = self.build_from_qasm("qaoa_barabasi_albert_N100_3reps.qasm")
+        self.dtc = self.build_from_qasm("dtc_100_cx_12345.qasm")
+        self.eoh = self.build_from_qasm("test_eoh_qasm.qasm")
```

Comment:
What are these two circuits?

Reply:
We should probably know what circuits we're benchmarking here to make sure it's cases we're interested in, since those are the metrics we want to optimize.

Reply:
In the current version of this PR, we have removed the usage of qasm benchmarks for this pass.